index.html

```diff
@@ -10325,26 +10331,29 @@ <h3><dfn>Release Actions</dfn></h3>
   is <a>no longer open</a>, return <a>error</a> with <a>error
   code</a> <a>no such window</a>.
 
- <li><p>Let <var>input state</var> be the result of <a>get the
- input state</a> with <a>session</a> and <a>current
- top-level browsing context</a>.
+ <li><p>Let <var>input state</var> be the result of <a>get the input state</a>
+  with <a>session</a> and <a>current top-level browsing context</a>.
 
  <li><p>Let <var>actions options</var> be a new <a>actions options</a>
   with the <a>is element origin</a> steps set to <a>represents a web
   element</a>, and the <a>get element origin</a> steps set
   to <a>get a WebElement origin</a>.
 
+ <li><p>Call <a>wait for an action queue token</a> with <var>input state</var>.
+
  <li><p>Let <var>undo actions</var> be <var>input
- state</var>&apos;s <a>input cancel list</a> in reverse order.
+  state</var>&apos;s <a>input cancel list</a> in reverse order.
 
- <li><p><a>Try</a> to <a>dispatch tick actions</a> with arguments
-  <var>undo
-  actions</var>, <code>0</code>,<var>session</var>&apos;s <a>current
-  browsing context</a>, and <a>actions options</a>.
+ <li><p><a>Dispatch actions</a> with <var>input state</var>,
+  <var>undo actions</var>, <a>current browsing context</a>,
+  and <var>actions options</var>. If this results in an <a>error</a>
+  return that error.
+
+ <li><p>Dequeue <var>input state</var>&apos;s <a>actions queue</a>.
+  <p>Assert: this returns <var>token</var>
```

Comment:
```suggestion
```

This already happens in `dispatch actions`, which also handles the failure case correctly.

Reply:
Oh, right. I missed that. Thanks.